update docs around sending chaff requests (#1234)
google/exposure-notifications-verification-server: docs/api.md

In addition to "real" requests, the server also accepts chaff (fake) requests.
These can be used to obfuscate real traffic from a network observer or server
operator. 

Chaff requests:

> * MUST resent the `X-API-Key` header with a valid API key (oterwise you will

Reviewer: ```suggestion
* MUST send the `X-API-Key` header with a valid API key (otherwise you will
```

>   get an authorized error)

Reviewer: ```suggestion
  get an unauthorized error)
```

> * MUST be sent via a `POST` requesxt, otherwise you will get an invalid method

Reviewer: ```suggestion
* MUST be sent via a `POST` request, otherwise you will get an invalid method
```

  error

> * SHOULD send a valid JSON body with padding out to the same size as the rest

Reviewer: ```suggestion
* SHOULD send a valid JSON body with random padding similar in size as the rest
```
